src/main/java/org/kiwiproject/eureka/EurekaServletHandler.java

```diff
@@ -164,7 +173,6 @@ private int getResponseCodeWithPossibleRetryFailure(ConcurrentHashMap<String, Pa
                 } else {
                     LOG.debug("Reached num times to fail {} for retry key {}, returning success code {}",
                             numTimesToFail, retryKey, successResponseCode);
-                    responseStatusCode = HttpServletResponse.SC_OK;
```

Comment:
Why did you remove this? Especially in light of the log message immediately above it. But it seems that removing it would cause problems, or am I missing something?

Reply:
The default is setting it to the success code.... this was ignoring the success code being passed in

Reply:
OK